pkg/webhook/clusterimagepolicy/clusterimagepolicy_types.go

```diff
@@ -80,6 +81,12 @@ type KeyRef struct {
 	// Data contains the inline public key
 	// +optional
 	Data string `json:"data,omitempty"`
+	// HashAlgorithm always default to sha256 if the algorithm hasn't been explicitly set
```

Comment:
I was wondering why we need both of these. I assume it's because we can't serialize HashAlgorithmCode below? If that's the case, maybe add a comment here (just like we do for PublicKeys below).

Reply:
Yes, I wanted to serialize the algo name in the configmap while having in the object the code so we can directly use it.